assets/lang/pt_BR.ini

```diff
@@ -937,7 +937,7 @@ Stopping.. = Parando...
 [Reporting]
 Bad = Ruim
 FeedbackCRCCalculating = CRC do Disco: Calculando...
-FeedbackCRCValue = CRC do Disco: %1
+FeedbackCRCValue = CRC do Disco: [VALUE]
```

Comment:
This should be %1 to fill in the right value.

Reply:
Thanks for the tip, didnt notice that when making the changes.